test/unit/cmd/http.test.ts

```diff
@@ -150,6 +150,23 @@ describe('http command', () => {
 			expect(mockedSocket.emit.lastCall.args[1]).to.have.nested.property('result.rawBody', expectedResult.result.rawBody);
 			expect(mockedSocket.emit.lastCall.args[1]).to.have.nested.property('result.rawHeaders', expectedResult.result.rawHeaders);
 		});
+
+		it('should ensure keepAlive header is disabled', () => {
```

Comment:
PR https://github.com/jsdelivr/globalping/pull/170 is changing the schema of this query. We might have to wait for it to merge, and then update the schema before adding this test before merging it.

Reply:
Sure, I'll watch the other PR and then adjust accordingly once that's merged 👍

Reply:
Oh it just got merged 👀

Reply:
:)